tests/10-vcdm2.js

```diff
@@ -299,8 +299,95 @@ describe('Types', function() {
 });
 
 // 4.5 Names and Descriptions https://w3c.github.io/vc-data-model/#names-and-descriptions
-// skipping tests for name and descrpition which are OPTIONAL properties
-// TODO: report that this section of the spec is being skipped and why
+// These tests for name and descrpition are OPTIONAL as those properties may
+// appear anywhere. However, we have tests for them (on `issuer` so far), so
+// keeping them in play seems prudent/useful. They can be expanded later also
+// to cover `name` and/or `description` anywhere they appear.
+describe('Names and Descriptions', function() {
+  setupMatrix.call(this);
+  for(const [name, implementation] of match) {
+    const endpoints = new TestEndpoints({implementation, tag});
+
+    describe(name, function() {
+      beforeEach(addPerTestMetadata);
+
+      const fixturePath = './input/names-and-descriptions';
```

Comment:
this is fine for now, but if this suite endpoints with a config dir making `fixturePath` configurable in the future is a good idea.